benchmark/test_runner/suite-tests.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const common = require('../common');
+const { finished } = require('node:stream/promises');
+
+const reporter = require('../fixtures/empty-test-reporter');
+
+const { describe, it } = require('node:test');
+
+const bench = common.createBenchmark(main, {
+  numberOfSuites: [10, 100],
+  testsPerSuite: [10, 100, 1000],
+}, {
+  // We don't want to test the reporter here
+  flags: ['--test-reporter=./benchmark/fixtures/empty-test-reporter.js'],
+});
+
+async function run(numberOfSuites, testsPerSuite) {
+  for (let i = 0; i < numberOfSuites; i++) {
+    describe(`${i}`, () => {
```

Comment:
```suggestion
    describe(`${i}`, { concurrency: false } , () => {
```
should probably test both true and false

Reply:
done, thanks